packages/smooth_app/pubspec.yaml

```diff
@@ -50,23 +51,13 @@ dependencies:
   percent_indicator: ^4.2.2
   mailto: ^2.0.0
   flutter_native_splash: ^2.1.6
-  google_mlkit_barcode_scanning: ^0.3.0
+  # Fork from cli1005 with iOS fix cf: https://github.com/openfoodfacts/smooth-app/issues/1155
+  google_ml_barcode_scanner:
+    git:
+      url: https://github.com/cli1005/google_ml_barcode_scanner.git
+      ref: master
   image_cropper: ^2.0.2
   auto_size_text: ^3.0.0
```

Comment:
```suggestion
  auto_size_text: ^3.0.0
  shared_preferences: ^2.0.15
  typed_data: ^1.3.0 # careful with 1.3.1 because of flutter_driver dependence
  intl: ^0.17.0
  flutter_isolate: ^2.0.2
  rxdart: ^0.27.3
  collection: ^1.16.0
  path: ^1.8.0 # careful with 1.8.1 because of https://github.com/flutter/flutter/issues/95478
  path_provider: ^2.0.10
  data_importer_shared:
    path: ../data_importer_shared
  data_importer:
    path: ../data_importer
  # Dependency used in data_importer, but as it's transitive, the only solution is to embed it here
  realm: 0.2.1+alpha
```